src/main/java/de/srendi/advancedperipherals/common/util/inventory/BasicCraftJob.java

```diff
@@ -0,0 +1,206 @@
+package de.srendi.advancedperipherals.common.util.inventory;
+
+import dan200.computercraft.api.lua.LuaFunction;
+import dan200.computercraft.api.peripheral.IComputerAccess;
+import de.srendi.advancedperipherals.common.util.StatusConstants;
+import net.minecraft.world.level.Level;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public abstract class BasicCraftJob {
+
+    protected static final String EVENT = "_crafting";
```

Comment:
```suggestion
    private static final String EVENT = "_crafting";
```

I don't think `protected static` will ever work.